README.md

```diff
@@ -8,14 +8,67 @@ This action allows caching dependencies and build outputs to improve workflow ex
 
 See ["Caching dependencies to speed up workflows"](https://help.github.com/github/automating-your-workflow-with-github-actions/caching-dependencies-to-speed-up-workflows).
 
+## What's New in V2
+
+* Added support for caching multiple paths,
+
+```yaml
+- name: Cache multiple relative paths
+  uses: actions/cache@v2
+  with:
+    path: |
+      node_modules
+      dist
+    key: ${{ runner.os }}-${{ hashFiles('**/lockfiles') }}
+```
+```yaml
+- name: Cache multiple absolute paths
+  uses: actions/cache@v2
+  with:
+    path: |
+      ~/cache
+      /path/to/dependencies
+    key: ${{ runner.os }}-${{ hashFiles('**/lockfiles') }}
+```
+
+[glob patterns](https://github.com/actions/toolkit/tree/master/packages/glob), 
+
+```yaml
+- name: Cache using glob patterns
+  uses: actions/cache@v2
+  with:
+    path: |
+      **/*.ts
+      **/node_modules
+    key: ${{ runner.os }}-${{ hashFiles('**/lockfiles') }}
+```
+
+and single file caches. 
+
+```yaml
+- name: Cache single file
+  uses: actions/cache@v2
+  with:
+    path: cache.tar
+    key: ${{ runner.os }}
+```
+
+* Increased perfomance and improved cache sizes using `zstd` compression
+> Note this feature is off for Windows runner that are using `bsdtar` (e.g., windows-latest hosted runner) due to a bug in ziping large random files with `bsdtar`
```

Comment:
```suggestion
> Note this feature is off for Windows runner that are using `bsdtar` (e.g., windows-latest hosted runner) due to a bug in zipping large random files with `bsdtar`
```

Should we link to the bsd tar issue?

Reply:
On this note, I'm not sure if we need to specify this caveat about Windows. The "What's New" doesn't need to be an exhaustive description of every feature. We could just say 
>Increased performance and improved cache sizes using `zstd` compression for Linux and macOS runners.

There's a very small portion of Windows runners that will benefit as well, but a majority will not.

>Should we link to the bsd tar issue?

The issue is https://github.com/actions/cache/issues/301. Linking the issue here in the What's New section would be good if we keep the note about this feature being disabled for most Windows runners.